openhands/server/auth.py

```diff
@@ -1,8 +1,25 @@
+from typing import Literal
+
 from fastapi import Request
 from pydantic import SecretStr
 
 
+def get_token(request: Request) -> SecretStr | None:
```

Comment:
we should define _both_ `request.state.github_token` and `request.state.gitlab_token`. A given user might attach both accounts.